query-parser/src/parse.rs

```diff
@@ -19,11 +19,12 @@ use self::mentat_query::{Element, FindSpec, Variable};
 
 use super::error::{FindParseError, FindParseResult};
 
-pub struct FindSp<I>(::std::marker::PhantomData<fn(I) -> I>);
+pub struct Find<I>(::std::marker::PhantomData<fn(I) -> I>);
 
-type FindSpParser<O, I> = Expected<FnParser<I, fn(I) -> ParseResult<O, I>>>;
+// Nothing about this is specific to the type of parser.
+type ResultParser<O, I> = Expected<FnParser<I, fn(I) -> ParseResult<O, I>>>;
```

Comment:
Mmm, we might want to lift this and share it for `tx-parser` as well.  Might not be worth the effort.